es/es-server/src/main/java/org/elasticsearch/Version.java

```diff
@@ -86,7 +86,7 @@
     public static final Version V_4_0_6 = new Version(ES_V_7_0_6_ID, ES_V_7_0_6_ID - INTERNAL_OFFSET, org.apache.lucene.util.Version.LUCENE_8_0_0);
     public static final Version V_4_0_7 = new Version(ES_V_7_0_7_ID, ES_V_7_0_7_ID - INTERNAL_OFFSET, org.apache.lucene.util.Version.LUCENE_8_0_0);
     public static final Version V_4_0_8 = new Version(ES_V_7_0_8_ID, ES_V_7_0_8_ID - INTERNAL_OFFSET, org.apache.lucene.util.Version.LUCENE_8_0_0);
-    public static final Version V_4_0_9 = new Version(ES_V_7_0_9_ID, ES_V_7_0_9_ID - INTERNAL_OFFSET, true, org.apache.lucene.util.Version.LUCENE_8_0_0);
+    public static final Version V_4_0_9 = new Version(ES_V_7_0_9_ID, ES_V_7_0_9_ID - INTERNAL_OFFSET, org.apache.lucene.util.Version.LUCENE_8_0_0);
```

Comment:
Do we need to create at this point V_4_0_10 snapshot?

Reply:
That will be a separate commit